### DISCUSSION TITLE
How can I get fanless on silent profile?

### DISCUSSION THREAD

**Discussion by moracabanas** [seerge/g-helper#286] [Q&A]
Thanks you so much for this work, after trying G14Control and having to keep Armory Crate constantly updating itself with no consent, this software pretty much solved my uncontrolled overheating on idle in 5 seconds with no instalation process. Pretty impresive!

Model: `GA401IV`
I tried to drop curve values to `off` but fans still running on 30ºC

**Comment by seerge** [comment]
@moracabanas did you click "Apply Custom fan curve" ? anyway without any extra info i can't say much. You may check log at ``%AppData%\GHelper`` and see if app accepts custom fan curve
